js/greeter.js

```diff
@@ -228,6 +228,8 @@ class AntergosTheme {
 
 			if ( lightdm[ cmd ] ) {
 				$( template ).appendTo( $( this.$actions_container ) ).click( action, ( event ) => {
+					// prevent accidential hibernates after resume 
```

Comment:
Looking at it more closely, this actually wouldn't do what its intended to do. I think the best thing to do is add the following to the end of the `initialize()` method:
```javascript
$( this.$actions_container ).children().blur();
```